packages/app/src/specs/SpecListGitInfo.vue

```diff
@@ -1,27 +1,31 @@
 <template>
   <div
-    class="h-full grid gap-9px grid-cols-[16px,auto] git-info-row items-center"
+    class="h-full grid gap-9px git-info-row items-center"
+    :class="{'grid-cols-[16px,auto]': classes.icon}"
     data-cy="git-info-row"
   >
     <Tooltip
+      v-if="classes.icon"
+      :key="props.gql?.statusType ?? undefined"
```

Comment:
It seems that the `Tooltip` component from `floating-vue` binds events to its contents when it is mounted, which we're losing when the contents change, causing the tooltip not to show up.
Adding the `key` attribute and causing it to change value when the target changes forces Vue to re-render the DOM, which allows for the events to be bound correctly. This may not be an ideal solution, but given the use case here it doesn't create much overhead as these changes are triggered by file status changes, which are not expected to be frequent within 60 seconds, which is the frequency used to update the file status icon.
[This article](https://michaelnthiessen.com/understanding-the-key-attribute) explains some more details about `key` in Vue. In summary:
> The key attribute tells Vue how your data relates to the HTML elements it's rendering to the screen. When your data changes, Vue uses these keys to know which HTML elements to remove or update, and if it needs to create any new ones.

Reply:
Yeah a `key` seems like the right move here 👍

Reply:
@ZachJW34 this was the fix for the issue you found